src/main/java/seedu/clirental/CliRental.java

```diff
@@ -0,0 +1,54 @@
+package seedu.clirental;
+
+import exceptions.CarException;
+import exceptions.CliRentalException;
+import exceptions.CustomerException;
+import parser.Parser;
+
+public class CliRental {
+    /**
+     * Main entry-point for the java.clirental.CliRental application.
+     */
+    public static void main(String[] args) {
+        printGreetings();
+        getName();
+
+        boolean isExit = false;
+
+        while (!isExit) {
+            try {
+                String action = Parser.getUserInput();
+
+                if (Parser.parse(action)) {
+                    isExit = true;
+                }
+
+            } catch (CustomerException exception){
+                exception.printErrorMessage();
+            } catch (NumberFormatException exception){
+                System.out.println("Unable to parse input");
+            } catch (CarException | CliRentalException e) {
+                System.out.println(e.getMessage());
+            }
+        }
+
+        System.out.println("Goodbye!");
+    }
+
+    public static void getName(){
```

Comment:
Readers would normally expect a Name return type for a method named getName.
Consider renaming this method.
This issue is apparent throughout the codebase.